doc/sphinx/proof-engine/tactics.rst

```diff
@@ -3427,6 +3427,8 @@ The general command to add a hint to some databases :n:`{+ @ident}` is
 
       No database name is given: the hint is registered in the core database.
```

Comment:
Should this read ```in the ``core`` database```?
It would be nice to make it clear which database is the default.